service/collector/nat.go

```diff
@@ -12,11 +13,13 @@ import (
 
 	clientaws "github.com/giantswarm/aws-operator/client/aws"
 	"github.com/giantswarm/aws-operator/service/controller/key"
+	"github.com/giantswarm/aws-operator/service/internal/cache"
 )
 
 const (
-	labelVPC = "vpc"
-	labelAZ  = "availability_zone"
+	awsNATlocker = "__awsNATlocker__"
```

Comment:
Can we have some comments why this is how it is?

Reply:
yeah sure, I am not sure if we have a convention for this, could not find in fmt repo